airflow/gcp/operators/pubsub.py

```diff
@@ -186,6 +264,13 @@ def __init__(
             fail_if_exists: bool = False,
             gcp_conn_id: str = 'google_cloud_default',
             delegate_to: Optional[str] = None,
+            push_config: Optional[Union[Dict, types.PushConfig]] = None,  # pylint: disable=no-member
+            retain_acked_messages: Optional[bool] = None,
+            message_retention_duration: Optional[Union[Dict, types.Duration]] = None,  # noqa E501 # pylint: disable=no-member,line-too-long
```

Comment:
I think that it would look better to import `PushConfig`, `Duration` from `types` and put pylint disable once.